src/style.css

```diff
@@ -2,0 +1,83 @@
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir.ttc');
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir.ttc');
+  font-weight: bold;
+}
+
+:root {
+  --font-family: Avenir, Arial, sans-serif;
+  --font-color: #3d4e61;
+  --background-color: #fff;
+  --border-color: #e1e7ed;
+  --box-shadow: rgba(61, 78, 97, 0.1);
+  --hover-box-shadow: rgba(61, 78, 97, 0.2);
+  --placeholder-color: #3d4e61;
+  --search-icon-size: 19px;
+  --search-icon-small-size: 11px;
+  --search-bar-height: 70px;
+  --search-bar-small-height: 42px;
+  --font-size: 16px;
+  --font-size-small: 14px;
+}
+
+body {
+  padding: 0;
+}
+
+.search-bar {
+  margin-top: 20px;
```

Comment:
Be consistent with your margins. Since you're adding a margin to the top, ensure that you're not adding margin to the bottom of any elements within `.search-bar` to avoid potential margin collapse issues.